src/main.rs

```diff
@@ -68,6 +68,8 @@ lazy_static! {
 
         config_dirs
     };
+
+    static ref CONFIG_T: AppConfig = AppConfig::get_config();
```

Comment:
Having it as a static reference here seems to be the only solution... That's not a problem right?

Reply:
Instead of making the config static, I think the less intrusive approach is to just check the config to see if option is enabled and pass the already lowercase version into `app_list_for_ext`.

ie. inside `open_file.rs`, change `_get_options` to take a `&AppConfig`

Reply:
Well, that works but just for the mimetype part, for theme and icons there is something more to do, but I'll try